Formula/curl.rb

```diff
@@ -22,24 +23,29 @@ class Curl < Formula
   keg_only :provided_by_macos
 
   depends_on "pkg-config" => :build
+  depends_on "libressl"
+  depends_on "nghttp2"
 
   uses_from_macos "zlib"
 
-  on_linux do
-    depends_on "openssl@1.1"
-  end
-
   def install
     system "./buildconf" if build.head?
 
+    libressl = Formula["libressl"]
     args = %W[
       --disable-debug
       --disable-dependency-tracking
       --disable-silent-rules
+      --disable-tls-srp
       --prefix=#{prefix}
-      --with-secure-transport
```

Comment:
Why no secure transport? (since macOS does have this by default)

Reply:
Incompatible with `HTTPS-proxy`